patch.c

```diff
@@ -80,11 +80,17 @@ int patch_execute(struct qdl_device *qdl, int (*apply)(struct qdl_device *qdl, s
 		return 0;
 
 	list_for_each_entry(patch, &patches, node) {
+		if(!patch->filename)
```

Comment:
please add a space after `if` keyword, like it's done everywhere in the code
```suggestion
		if (!patch->filename)
```